src/main/java/org/jenkinsci/plugins/googleplayandroidpublisher/ApkPublisher.java

```diff
@@ -55,7 +55,7 @@
 public class ApkPublisher extends GooglePlayPublisher {
 
     @DataBoundSetter
-    private String apkFilesPattern;
+    protected String apkFilesPattern;
```

Comment:
I'm not really sure the best way to allow the tests to set these inputs, and I couldn't find any existing built-in (to Jenkins/tests) way to provide data for the `@DataBoundSetter` fields.

When the fields are `protected`, unit tests are allowed to access and set those fields directly, so that's what I went with.